### PR TITLE
Fix charging initialisation fee for an initialised account

### LINKED ISSUE #1727: Charging initialisation fee for an initialised account
### Expected behavior

- Initialisation fee should only be charged for uninitialised account
- Add an unit/e2e test to test initialised uninitialised account scenario
- Send token transfer to an uninitialised account should add the initialisation fee

### Actual behavior

Charging initialisation fee for an initialised account

### Steps to reproduce

- Send token transfer to an initialised account

### Which version(s) does this affect? (Environment, OS, etc...)
3.0.0

### PR DESCRIPTION
**Pull request by ClementeSerrano**
### What was the problem?

This PR resolves #1727

### How was it solved?

- [x] Fetch extra command fee dynamically based on recipientAddress and tokenID change on token:transfer form.
- [x] Disable initialisation fee query when recipientAddress and tokenID are not selected on
token:transfer form. Allow refetch query to trigger dynamically on input changes.

https://user-images.githubusercontent.com/9727036/228265215-6608f9ec-f5c4-4f3d-84cb-824c887a6a4b.mp4

### How was it tested?

- [x] iOS Simulator.
- [x] Android Emulator.

### CODE REVIEW

File: src/modules/SendToken/hooks/useSendTokenForm.js [hunk 1]
```diff
@@ -78,14 +79,23 @@ export default function useSendTokenForm({ transaction, isTransactionSuccess, in
 
   const recipientApplicationChainID = form.watch('recipientApplicationChainID');
   const senderApplicationChainID = form.watch('senderApplicationChainID');
+  const recipientAddress = form.watch('recipientAccountAddress');
+  const tokenID = form.watch('tokenID');
 
-  const isCrossChainTransfer = senderApplicationChainID !== recipientApplicationChainID;
+  const isCrossChainTransaction = senderApplicationChainID !== recipientApplicationChainID;
```
Comment (ManuGowda) [inline src/modules/SendToken/hooks/useSendTokenForm.js:85]: cross chain transfer was more relevant?

**Review by ManuGowda — COMMENTED** [review]
The changes looks mostly ok, ill assign @Balanced02 so that he can test on both iOS and android

File: src/modules/SendToken/hooks/useSendTokenForm.js [hunk 2]
```diff
@@ -78,14 +79,23 @@ export default function useSendTokenForm({ transaction, isTransactionSuccess, in
...
 
   const { data: recipientApplicationChainChannelData } = useChainChannelQuery(
```
[the remark this answers is not in the document]
Comment (ClementeSerrano) [reply src/modules/SendToken/hooks/useSendTokenForm.js:87]: `useChainChannelQuery` is used to invoke `interoperability_getChannel`, which is necessary for getting the `messageFeeTokenID` for crosschain transfers

File: src/modules/SendToken/hooks/useSendTokenForm.js (hunk 1 above)
Comment (ClementeSerrano) [reply src/modules/SendToken/hooks/useSendTokenForm.js]: ok, renamed it back

**Pull request merged**